_posts/en/podcast/2023-04-06-newsletter-recap.md

```diff
@@ -18,6 +18,618 @@ Mark "Murch" Erhardt and Mike Schmidt are joined by Sergi Delgado Segura to disc
+subscription is expired, then there's actually some grace period, and so on,
+just in case the user forgets, but at some point, the tower will delete the
+data; I mean the time you have to complain about something, if you work with
+accumulators.  If you don't work with accumulators, you actually have data that
+you can prove for life, let's say.
+
+But if you're working with accumulators, the tower is the one that has to end up
+providing the data that they are supposed to be hosting, so at some point if you
+try to say, "Hey, you cheated me three years ago", it's like, "Yeah, I don't
+have the data anymore, so good luck trying to prove that".  That doesn't happen
+if you're not using accumulators, because the client may have all the data
+necessary to make this work, but it means that they also have to store way more
+data on their end, which we are trying to prevent, or improve.
+
+**Mark Erhardt**: Understood.
+
+**Mike Schmidt**: How has feedback been from the mailing list and elsewhere on
+your initial post and discussions you've had, either on the mailing list or
+outside of that?
+
+**Sergi Delgado Segura**: So, I got feedback from both Calvin and Salvatore
+since the very first day I was looking into this.  I pinged them directly before
+sending the email.  I don't think anyone has replied to the mailing list post,
+but as I was saying before, not many people are interested in watchtowers at the
+development level, so that's usual.  I'm checking the email just in case someone
+has replied and I haven't seen, but I don't think so.  It feels like everyone
+has a lot on their plates and if you don't ping them directly, or it's something
+that is related to their work, you may not get a reply.  But hey, listener, if
+you're listening to this and you're interested in watchtowers or accumulators,
+yeah, ping me, let's have a chat.
+
+**Mike Schmidt**: Great call to action.  Anything else you'd say before we wrap
+up on this news item?
+
+**Sergi Delgado Segura**: No, not really from my end.  I think we've covered
+most of what the proposal is about.  If anyone is more intrigued about how it
+is, there's a link on both the mailing list mail and the Optech newsletter.
+There's a gist about the proposal itself, so it's not that long to read.  I
+think you can read it in like ten minutes and understand it pretty
+straightforward.  So, yeah, give me a ping, or whatever.
+
+**Mike Schmidt**: Murch, anything before we move on?
+
+**Mark Erhardt**: No, I think that's all.
+
+**Mike Schmidt**: Well, thank you for joining us.  You're welcome to stay on, we
+have a bunch of Lightning-related PRs later in the newsletter if you care to
+comment on any of those.  But otherwise, if you're busy with other things,
+you're free to drop off as well.
+
+**Sergi Delgado Segura**: I'll stay around just in case.
+
+**Mike Schmidt**: Awesome.  Next section of the newsletter is releases and
+release candidates.
+
+{:#lnd}
```

Comment:
These ids for the titles of the transcript's sections are not used based on the new logic.

Reply:
Absolutely, my oversight. Fixed.